.gitignore

```diff
@@ -25,3 +25,6 @@ erl_crash.dump
 # Ignore package tarball (built via "mix hex.build").
 optimal-*.tar
 
+# IDEs
+*.iml
```

Comment:
These should be added to your global gitignore, not added to each project that you use.